docker/Dockerfile

```diff
@@ -35,7 +35,31 @@ RUN conda config --set show_channel_urls true && \
     conda install --update-all --force-reinstall --yes --file /tmp/requirements.txt && \
     conda clean --all --yes && \
     conda info --all && \
-    conda list
+    conda list && \
+    # install tools for using OpenStreetMap
+    apt-get update && apt-get install -y osm2pgsql osmctools && \
+    # Install tinytex, a minimal TeX distribution for making pdf documentation
+    apt-get update && apt-get install -y perl wget libfontconfig1 && \
+    wget -qO- "https://yihui.org/tinytex/install-bin-unix.sh" | sh  && \
+    apt-get clean && \
+    # Install LaTeX packages
+    /root/bin/tlmgr install xetex xcolor pgf fancyhdr parskip babel-english  \ 
+    units lastpage mdwtools comment   fontawesome times     \
+    fncychap titlesec tabulary varwidth wrapfig capt-of needspace  \
+    polyglossia fontspec cmap gnu-freefont upquote oberdiek latexmk \
+    caption makecell multirow changepage \
+    --repository=http://mirror.ctan.org/systems/texlive/tlnet \
+    && /root/bin/fmtutil-sys --all && \
+    # Install additional LaTeX packages
+    # installing seperately to avoid error associated with multirow && \
+    # install the make build tools, for compiling sphinx documentation
+    apt-get update && apt-get install -y make   && \
+    # install the contextily package using pip, required for validation report basemaps
+    RUN pip install contextily
```

Comment:
One last thing... `contextily` is available via conda-forge: https://anaconda.org/conda-forge/contextily Can we just add this to `requirements.txt` instead of having a separate pip install just for it?

If not, can we remove the `RUN` part here so `pip install` operates within the single `RUN` command above?

Reply:
Thanks Geoff -- contextily is written in the requirements.txt, but it seems it is nevertheless required to be installed using pip; when I didn't do this, it asked for the package to be installed (and once installed using pip, it worked).  I accidentally missed removing that RUN command when concatenating the build into a single run --- that's now fixed and pushed.  Cheers,  Carl